content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -220,6 +221,10 @@ of this approach is a small container running logrotate periodically.
 However, it's recommended to use `stdout` and `stderr` directly and leave rotation
 and retention policies to the kubelet.
 
+{{< note >}}
+This works because the sidecar will be restarted until the file exists. For more advanced usage have a look to `ls app.log && tail -F app.log` is more powerful for legacy application doing daily rolling.
```

Comment:
I'm not sure this is technically accurate.

I take “a small container running logrotate periodically” to mean one container with a supervisor that stays running. The supervisor regularly forks a child process and execs `logrotate`. In the same Pod, there is a different container running the workload; the logrotate container is a sidecar.

At the moment I think I prefer the existing page; I don't see this proposed change as an improvement.

Reply:
The use case I try to document is when the main container/app is managing the lifecycle of the log. The sidecar is only here to expose to stdout without affecting the file. It is the case of most java application. This Tip & Trick is very useful for migration purpose. Since we cannot change the main application code easily we had to find a workaround and this one works.

* `ls` part is hear to wait for the file to exist before to receive traffic
* `tail -F` is there when the application roll the file

Do you need a working example ?

Reply:
This is a case where a small suggestion triggers a bigger change. To add more commentary specifically on the _Streaming sidecar container_ example, I think it'd only make sense to:
- migrate that example to a _new_ task page
- add the commentary there.

So, to get a few lines of note added, you need to do a bunch of not closely related changes. That's how I'd tackle this.

Reply:
sound good to me. Any insight about where to add this ? 

https://github.com/kubernetes/website/tree/master/content/en/docs/tasks
I am thinking of `configure-pod-container`, `debug-application-cluster` or  `administer-cluster`

Reply:
Definitely `debug-application-cluster`, which in English has the friendly title of “Monitoring, Logging, and Debugging”

Reply:
last question. do you prefer me to open a new PR or do everything here?

Reply:
Whichever works for you!

You can use the `/retitle` Prow command if you like and you can also edit the PR description.

Reply:
I am working on it, I will put the new PR here once it's open, and close this one after.

Reply:
@NargiT how did that go? Maybe I can help?